src/NotificationsPage/components/UnreadNotifications.js

```diff
@@ -37,35 +37,56 @@ class UnreadNotification extends Component {
+                          onClick={() => this.props.markOneAsRead(notification.sent_to, notification.id)}
+                          method="POST"><p
+                          className="connection-name">{notification.sent_from_name} </p></Link>
+                    <span>has accepted your connection request</span>
+                  </div>
+                </div>
               </div>
             </div>
           );
         }
         if (notification.action === "connection_request" && notification.seen == null) {
           return (
             <div key={notification.id}>
-              <div id="user-info">
-                <img id="connection-img" src="https://u.o0bc.com/avatars/no-user-image.gif" alt=""/>
-
-                <Link to={`/profile/${notification.sent_from}`}
-                      onClick={() => this.props.markOneAsRead(notification.sent_to, notification.id)}
-                      method="POST"><p
-                    	className="connection-name">{notification.sent_from_name}</p></Link>
-                <span>has sent you a connection request</span>
-                <button type="button" className="btn-accept-decline"
-                        onClick={() => this.acceptTheConnection(userId, notification.sent_from, callback, callback2, notification.sent_to, notification.id)}>accept
-                </button>
-                <button type="button" className="btn-accept-decline"
-                        onClick={() => this.declineTheConnection(userId, notification.sent_from, callback, notification.sent_to, notification.id)}>decline
-                </button>
+              <div id="user-info" classname="container" >
+                <div className="row">
+                  <div className="col-xs-9">
+                    <img id="connection-img" src="https://u.o0bc.com/avatars/no-user-image.gif" alt=""/>
+
+                    <Link to={`/profile/${notification.sent_from}`}
+                          onClick={() => this.props.markOneAsRead(notification.sent_to, notification.id)}
+                          method="POST"><p
+                          className="connection-name">{notification.sent_from_name}</p></Link>
+                    <span>has sent you a connection request</span>
+                  </div>
+                  <div className="col-xs-3 btn-container">
+                    <button type="button" className="btn-accept-decline"
+                            onClick={() => this.acceptTheConnection(userId, notification.sent_from, callback, callback2, notification.sent_to, notification.id)}>accept
+                    </button>
+                    <button type="button" className="btn-accept-decline"
+                            onClick={() => this.declineTheConnection(userId, notification.sent_from, callback, notification.sent_to, notification.id)}>decline
+                    </button>
+                  </div>
+                </div>
+
+
+
+
+
+
+
+
+
+                
+  
```

Comment:
Have a few of these huge chunks of newlines, can we clean these up here and in a couple other places?